package.json

```diff
@@ -36,6 +36,7 @@
     "build-dev": "webpack --config webpack.dev.js",
     "watch": "webpack --config webpack.dev.js --watch",
     "lint": "eslint . --ignore-path .gitignore --ext .js,.jsx,.ts,.tsx",
+    "fix": "npm run lint -- --fix",
```

Comment:
I think we don't want to run fix by default, as we want the build to fail when committing code that doesn't pass the linting

Reply:
yes I wasn't planning on adding this to the build, it's just so I don't have to remember the weird syntax

Reply:
👍